### DISCUSSION TITLE
How to make global shortcut blank?

### DISCUSSION THREAD

**Discussion by zycin1** [darkreader/darkreader#11308] [Q&A] [answered]
![Screenshot 2023-05-23 093326](https://github.com/darkreader/darkreader/assets/59542219/a37cd793-0e86-4bb2-8f8e-d586b5df2655)

**Comment by alex98721** [comment, marked as answer]
I use Firefox, so I can tell you that if you're using the same as me you can change it by clicking the **jigsaw puzzle piece icon** on the top right to toggle your list of extensions, then select **Manage extensions** followed by clicking on the **gear icon** at the top right of the list your given and finally selecting **Manage Extension Shortcuts** which should take you to the menu where you can configure any number of allowable shortcuts for your currently installed extensions. To get rid of a shortcut and leave it blank, pressing the **trash can icon** should do the trick and then you can just leave it like that. I'm pretty sure what I've just described is a rather similar process on Chrome too and perhaps other browsers follow similar conventions.

**Reply by zycin1** [reply]
Yes, that was it, thank you :D
